library/src/scala/NamedTuple.scala

```diff
@@ -128,6 +128,10 @@ object NamedTuple:
   /** The empty named tuple */
   val Empty: Empty = EmptyTuple
 
+  /** The ordering instance for named tuples */
+  given namedTupleOrdering: [N <: Tuple, V <: Tuple] => (ord: Ordering[V]) => Ordering[NamedTuple[N, V]]:
+    def compare(x: NamedTuple[N, V], y: NamedTuple[N, V]): Int =
+      ord.compare(x.toTuple, y.toTuple)
```

Comment:
Nitpick, but is it worth constructing this manually, when there's already a `Ordering.by` helper for situations like this?

```scala
given namedTupleOrdering: [N <: Tuple, V <: Tuple] => (Ordering[V]) => Ordering[NamedTuple[N, V]] =
  Ordering.by(_.toTuple)
```